src/commander/commander.c

```diff
@@ -264,10 +264,12 @@ static commander_error_t _api_process(const Request* request, Response* response
     case Request_btc_pub_tag:
         response->which_response = Response_pub_tag;
         return commander_btc_pub(&(request->request.btc_pub), &(response->response.pub));
+#if defined(APP_BTC)
```

Comment:
move one case up?

should be `defined(APP_BTC) || defined(APP_LTC)`? Otherwise `APP_LTC` without `APP_BTC` does not work. Then more granular filtering is needed in the relevant functions.

Reply:
What about ifdefing the relevant includes at the top of the file too?

Reply:
fixed both